jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -444,9 +443,6 @@ else if (_parser.inContentState() && _generator.isPersistent())
 
         // Not in a race here with onFillable, because it has given up control before calling handle.
         // in a slight race with #completed, but not sure what to do with that anyway.
```

Comment:
Is this comment no longer relevant as we moved the chunk release?

Reply:
It is still kind of relevant, but wrong because method names have changed.  I'll remove because it's a negative anyway.